### ISSUE TITLE
Footer Flow e2e testing - gctools feature accessibility check updates

### ISSUE THREAD

**Issue by AdiMakkar** [gctools-outilsgc/gccollab-frontend#247]
Based on the technical debt in the footer axe accessibility e2e testing, the gctools page that Git Hub hosts are not accessible. The page must not be axe checked because the axe test will keep failing and nothing can be done since we don't own GitHub. 

The solution is being implemented to remove the GitHub test altogether. 

refer to #249
